src/components/global-logout/global-logout-controller.ts

```diff
@@ -0,0 +1,19 @@
+import { Request, Response } from "express";
+import { eventService } from "../../services/event-service";
+import { EventName, LogoutState } from "../../app.constants";
+import { handleLogout } from "../../utils/logout";
+
+export function globalLogoutGet(req: Request, res: Response): void {
+  res.render("global-logout/index.njk", {});
+}
+
+export async function globalLogoutPost(req: Request, res: Response) {
+  const service = eventService();
+  const auditEvent = service.buildAuditEvent(
+    req,
+    res,
+    EventName.HOME_GLOBAL_LOGOUT_REQUESTED
+  );
+  service.send(auditEvent, res.locals.trace);
```

Comment:
Hi @saralk - could we add the following just before `await handleLogout`

```
  req.metrics?.addMetric("globalLogoutPost", MetricUnit.Count, 1);
```
  
and also do a refactor in `backchannel-logout-controller` and rename the metric in there to `backChannelLogoutPost` instead of `globalLogoutPost`